### PR TITLE
Add Input css

### PR DESCRIPTION
**Pull request by rahulcs754**
image

Add Input :
Input textbox with validation and errors style

### CODE REVIEW

File: css/components/input.css [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    border-radius: 0.25rem;
+    padding: 0.3rem 0.5rem;  
+}
+
+.field-item:focus{
+    border-color:var(--light-blue);
+    box-shadow:0 0 4px 0 var(--light-blue);
+}
+
+.label{
```
Comment (tanishq20) [inline css/components/input.css]: Avoid giving class name same as tag name.

**Review by tanishq20 — COMMENTED** [review]
Suggested small change.

File: css/components/input.css (hunk 1 above)
Comment (rahulcs754) [reply css/components/input.css]: as you suggest to me. i changed

**Review by tanishq20 — APPROVED** [review]

**Review by shuntzhere — APPROVED** [review]
Looks good for the merge

**Pull request merged**